src/data_provider/CMakeLists.txt

```diff
@@ -179,6 +191,13 @@ endif(CMAKE_SYSTEM_NAME STREQUAL "Windows")
 
 target_link_libraries(sysinfo wazuhext)
 
+if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+  target_link_libraries(sysinfo urlrequest)
+  if(CMAKE_CHECK_CENTOS5)
+      set(USE_HTTP 1)
+  endif(CMAKE_CHECK_CENTOS5)
+  add_subdirectory(${SRC_FOLDER}/shared_modules/http-request ${SRC_FOLDER}/shared_modules/http-request/build)
```

Comment:
We need this due in case of absolute source path (and when this path isn't in your source tree), CMake cannot guess build directory, and you need to provide it explicitly.

Reply:
Isn't this a case for include_directories()?

Reply:
Thank you for clarifying. 
So I understand now that you are specifying a binary dir

https://cmake.org/cmake/help/latest/command/add_subdirectory.html

> add_subdirectory(source_dir [binary_dir] [EXCLUDE_FROM_ALL] [SYSTEM])

Reply:
I think is not possible @MiguelazoDS , because the function `include_directories()` is used to add the given directories to those the compiler uses to search for include files.

https://cmake.org/cmake/help/latest/command/include_directories.html